ci/ray_ci/test_linux_tester_container.py

```diff
@@ -153,5 +165,20 @@ def _mock_shard_tests(tests: List[str], workers: int, worker_id: int) -> List[st
         assert not container.run_tests(["bad_test"], [])
 
 
+def test_create_bazel_log_mount() -> None:
+    with tempfile.TemporaryDirectory() as tmpdir, mock.patch(
+        "random.choice",
+        return_value="a",
```

Comment:
maybe just allow sending in an id explicitly? and when it is None/empty, generate a random one?